infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/workflows/RemoveNodeWorkflow.java

```diff
@@ -0,0 +1,74 @@
+package org.corfudb.infrastructure.orchestrator.workflows;
+
+import com.google.common.collect.ImmutableList;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.google.common.collect.ImmutableList'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)